common-travel.lic

```diff
@@ -8,6 +8,82 @@ custom_require.call(%w[common common-arcana events drinfomon])
 module DRCT
   module_function
 
+  def buy_item(room, item)
+    walk_to(room)
+
+    # Amount to pay should be first capturing group.
+    patterns = [/prepared to offer it to you for (.*) (?:kronar|lirum|dokora)s?/,
+                /Let me but ask the humble sum of (.*) coins/,
+                /for a (?:mere )?(.*) (?:kronar|lirum|dokora)s?/,
+                /I can let that go for...(.*) (?:kronar|lirum|dokora)s?/,
+                /cost you (?:just )?(.*) (?:kronar|lirum|dokora)s?/,
+                /it may be yours for just (.*) (?:kronar|lirum|dokora)s?/,
+                /I'll give that to you for (.*) (?:kronar|lirum|dokora)s?/,
+                /I ask that you give (.*) copper (?:kronar|lirum|dokora)s?/,
+                /it'll be (.*) (?:kronar|lirum|dokora)s?/,
+                /the price of (.*) coins? is all I ask/,
+                /tis only (.*) (?:kronar|lirum|dokora)s?/,
+                /That will be (.*) copper (?:kronar|lirum|dokora)s? please/,
+                /to you for (.*) (?:kronar|lirum|dokora)s?/,
+                /I ask (.*) copper (?:kronar|lirum|dokora)s or if you would prefer/,
+                /Cost you just (.*) (?:kronar|lirum|dokora)s?, okie dokie\?/i,
+                /It will cost just (.*) (?:kronar|lirum|dokora)s?/i,
+                /I would suggest (.*) (?:kronar|lirum|dokora)s?/i,
+                /to you for (.*) (?:kronar|lirum|dokora)s?/i,
+                /asking (.*) (?:kronar|lirum|dokora)s?/i,
+                'You decide to purchase',
+                'Buy what']
+
+    match = DRC.bput("buy #{item}", *patterns)
+    if match
+      patterns.each { |p| break if p.match(match) }
+    end
+    amount = Regexp.last_match(1)
+
+    fput("offer #{amount}") if amount
+  end
+
+  def order_item(room, item_number)
+    walk_to(room)
+
+    return if DRC.bput("order #{item_number}", 'Just order it again', 'you don\'t have enough coins') == 'you don\'t have enough coins'
+    DRC.bput("order #{item_number}", 'takes some coins from you')
+  end
+
+  def dispose(item, trash_room = nil)
+    walk_to(trash_room) unless trash_room.nil?
+    case DRC.bput("get my #{item}", 'You get', 'What were you', 'You pick up', 'You are already holding that')
+    when 'You get', 'You pick up', 'You are already holding that'
+      DRCI.dispose_trash(item)
```

Comment:
Should `common-items` be added to this file's "require" statement at top?

Reply:
Yep. Good catch. Haven't had it fail in my testing yet, but as we know, that's just a fun order of loading happenstance.